src/test/java/uk/nhs/digital/nhsconnect/lab/results/sequence/SequenceRepositoryTest.java

```diff
@@ -0,0 +1,52 @@
+package uk.nhs.digital.nhsconnect.lab.results.sequence;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.data.mongodb.core.FindAndModifyOptions;
+import org.springframework.data.mongodb.core.MongoOperations;
+import org.springframework.data.mongodb.core.query.Query;
+import org.springframework.data.mongodb.core.query.Update;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+@SuppressWarnings("checkstyle:magicnumber")
+class SequenceRepositoryTest {
```

Comment:
Can you create a test for the `getNextForTransaction` method in `SequenceRepository` too? I don't think it's covered

Reply:
Done